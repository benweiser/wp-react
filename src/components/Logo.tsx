import * as React from "react";
import styled from "styled-components";

const logo = require("../assets/images/logo.png");

const Logo: React.SFC<{}> = (): JSX.Element => {
  return (
    <div>
      <StyledImage src={logo} alt="My Logo" />
      <span>Ben Weiser</span>
    </div>
  );
};

const StyledImage = styled.img`
  border-radius: 100px;
  max-width: 75px;
  padding: 4px;
  border: 1px solid #ccc;
`;

export default Logo;
