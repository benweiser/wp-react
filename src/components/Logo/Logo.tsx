import * as React from "react";
import styled from "styled-components";

const logo = require("../../assets/images/logo.png");

const StyledImage = styled.img`
  border-radius: 100px;
  max-width: 75px;
  padding: 4px;
  border: 1px solid #ccc;
`;

const Logo: React.SFC<{}> = (): JSX.Element => {
  return (
    <div>
      <StyledImage src={logo} alt="My Logo" />
      <span>Ben Weiser</span>
    </div>
  );
};

export default Logo;
