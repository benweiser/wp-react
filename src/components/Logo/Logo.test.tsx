import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Logo from "./Logo";

it("Logo component renders correctly", () => {
  const wrapper = shallow(
      <Logo/>
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
