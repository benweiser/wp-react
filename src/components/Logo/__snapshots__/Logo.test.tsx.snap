// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders correctly 1`] = `
<div>
  <styled.img
    alt="My Logo"
    src="logo.png"
  />
  <span>
    Ben Weiser
  </span>
</div>
`;
